### DISCUSSION TITLE
What can this be used for?

### DISCUSSION THREAD

**Discussion by e-t-l** [iamr0s/Dhizuku#26] [Q&A]
I'm confused, what is the benefit of giving different apps device owner privilege? Can anyone share an example of how they're currently using Dhizuku (and how it's different from using Shizuku)?

**Comment by iamr0s** [comment]
Install silent, disable Hardware function and more...

**Reply by sgitsp** [reply]
Yeah.. been tested on [雹 Hail](https://github.com/aistra0528/Hail) and you've done a great job! Maybe it would be better if Dhizuku could support more apps.

By the way, someone should start making a list of supported apps like in the [awesome-shizuku](https://github.com/ThePBone/awesome-shizuku#awesome-shizuku) lists 👍

**Reply by Bluscream** [reply]
I would agree with you there, i'm not quite sure if i can use my device owners (shizuku is one of them)

**Comment by nktnet1** [comment]
To explore all of the capabilities of a Device Owner app, I suggest looking into:
- Google's [Test DPC](https://github.com/googlesamples/android-testdpc) (requires [Dhizuku API Exposed](https://github.com/iamr0s/Dhizuku-API-Xposed))
- [OwnDroid](https://github.com/BinTianqi/OwnDroid) (supports Dhizuku natively)

Personally, for an app I'm developing ([Webview Kiosk](https://github.com/nktnet1/webview-kiosk/discussions)), I make use of a feature called [Lock Task Mode](https://developer.android.com/work/dpc/dedicated-devices/lock-task-mode) which locks the device to a single app (no changing apps, messing with status bar, etc).

Other features which I leverage [OwnDroid](https://github.com/BinTianqi/OwnDroid) for are:
- managed configurations (app restrictions)
- block uninstall
- user restrictions (e.g. block changes to wifi/bluetooth/location)
